cloud/docker.go

```diff
@@ -74,8 +74,11 @@ func (m *dockerManager) SpawnHost(ctx context.Context, h *host.Host) (*host.Host
 
 	// Create container
 	if err = m.client.CreateContainer(ctx, parentHost, h); err != nil {
-		err = errors.Wrapf(err, "Failed to create container for host '%s'", hostIP)
-		grip.Error(err)
+		err = errors.Wrapf(err, "Failed to create container for host '%s'", h.Id)
+		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
Are these really errors (in the sense of indicating something wrong with the Evergreen application), or should they be logged at a lower level, since they indicate maybe something flaky with Docker?